### PR TITLE
Added browser support table

### PR DESCRIPTION
**Pull request by rfreitas**
# Problem
Clients didn't have visibility on which browsers we support.

# Solution
Added it to the doc

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a ] Have new automated tests been implemented?
- [ n/a] Have new manual tests been written down?
- [ n/a] Have tests passed locally?
- [ n/a] Have any new strings been translated?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,6 +339,13 @@ A few things to check before you go live:
 - Make sure you have set up webhooks to receive live events
 - Make sure you have entered correct billing details inside your [Onfido Dashboard](https://onfido.com/dashboard/)
 
+## Browser support
+
+![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png) | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png) | ![IE](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png) | ![Opera](https://raw.githubusercontent.com/alrra/browser-logos/master/src/opera/opera_48x48.png) | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png)
+--- | --- | --- | --- | --- |
+Latest ✔ | Latest ✔ | 11+ ✔ | Latest ✔ | 6.1+ ✔ |
```
Comment (dawiddylowicz) [inline README.md]: Are we sure to support such a low Safari version? As far as I remember, the previous version didn't provide support for webcam

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-304-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://304-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md (hunk 1 above)
Comment (seewah) [reply README.md]: yeah why dont we just go for latest safari, like chrome and FF? Isnt safari one of those "evergreen" browsers?

Comment (rfreitas) [reply README.md]: I think you have to update it on the app store.
But I'm okay with bumping it up.

Comment (rfreitas) [reply README.md]: btw: https://www.netmarketshare.com/browser-market-share.aspx?options=%7B%22filter%22%3A%7B%22%24and%22%3A%5B%7B%22deviceType%22%3A%7B%22%24in%22%3A%5B%22Desktop%2Flaptop%22%5D%7D%7D%5D%7D%2C%22dateLabel%22%3A%22Trend%22%2C%22attributes%22%3A%22share%22%2C%22group%22%3A%22browserVersion%22%2C%22sort%22%3A%7B%22share%22%3A-1%7D%2C%22id%22%3A%22browsersDesktopVersions%22%2C%22dateInterval%22%3A%22Monthly%22%2C%22dateStart%22%3A%222017-01%22%2C%22dateEnd%22%3A%222017-12%22%2C%22segments%22%3A%22-1000%22%7D

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-304-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-304-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://304-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-304-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-304-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-304-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-304-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**